### PR TITLE
Remove frontmatter legacy variables

### PR DESCRIPTION
**Pull request by patriciabarrosssw**
PBIs 
#48710 https://ssw.visualstudio.com/SSW.People/_workitems/edit/48710
#48093 https://ssw.visualstudio.com/SSW.People/_workitems/edit/48093
#50679 https://ssw.visualstudio.com/SSW.People/_workitems/edit/50679

### CODE REVIEW

File: Calum-Simpson/Calum-Simpson.md [hunk 1]
```diff
@@ -1,20 +1,9 @@
 ---
-currentEmployee: true
-id: "CalumSimpson"
-name: "Calum Simpson"
-nickname: ""
+id: "6e4242c5-6414-e911-81cd-00155d012c51"
 
 category: "Developers"
 role:  "Senior Software Architect	"
```
Comment (christianmorfordwaitessw) [inline Calum-Simpson/Calum-Simpson.md:5]: remove extra white space

File: David-Cai/David-Cai.md [hunk 2]
```diff
@@ -1,20 +1,10 @@
 ---
-currentEmployee: false
-id: "DavidCai"
+id: DavidCai"
```
Comment (christianmorfordwaitessw) [inline David-Cai/David-Cai.md:2]: Missing quotation mark in id field

File: Calum-Simpson/Calum-Simpson.md (hunk 1 above)
Comment (patriciabarrosssw) [reply Calum-Simpson/Calum-Simpson.md]: done

File: David-Cai/David-Cai.md (hunk 2 above)
Comment (patriciabarrosssw) [reply David-Cai/David-Cai.md]: done

File: Robert-Jia/Robert-Jia.md [hunk 3]
```diff
@@ -1,20 +1,9 @@
 ---
-currentEmployee: false
-id: "RobertJia"
-name: "Robert Jia"
-nickname: ""
+id: "a6b85481-0d6e-e311-8103-00783af55a15"
 
 category: "Developers"
 role:  "Software Developer "
```
Comment (christianmorfordwaitessw) [inline Robert-Jia/Robert-Jia.md:5]: remove extra whitespace on role

**Pull request merged**